backend/codesirius/internal_api/views/hidden_test_bundle.py

```
@@ -0,0 +1,51 @@
+from internal_api.serializers.hidden_test_bundle import HiddenTestBundleSerializer
+from problems.models import HiddenTestBundle, Problem
+
+logger = logging.getLogger(__name__)
+
+
+class HiddenTestBundleAPIView(APIView):
+    def get_problem(self, problem_pk: int) -> Problem:
+        try:
+            return Problem.objects.get(id=problem_pk)
+        except Problem.DoesNotExist:
+            raise NotFound("Problem not found")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)